bubblewrap.c

```diff
@@ -1104,9 +1104,8 @@ privileged_op (int         privileged_op_socket,
 
         /* If we are given a size for the tmpfs, validate that it doesn't
          * contain any special characters. */
-        const size_t arg2_len = strlen (arg2);
-        if (arg2_len > 0) {
-          for (size_t i = 0; i < arg2_len; i++) {
+        if (arg2 != NULL && arg2[0] != '\0') {
```

Comment:
`--size: Properly manage nullity of op->size` seems like it should be part of the commit with the initial implementation, rather than a later fix.